src/site/content/en/fast/codelab-preload-critical-assets/index.md

```diff
@@ -193,8 +193,8 @@ when needed.
 ## Preloading and prefetching with webpack
 
 The
-[code splitting guide](/reduce-javascript-payloads-with-code-splitting)
-explores the use of dynamic imports to split a bundle into multiple chunks.
+[Reduce JavaScript payloads with code-splitting](/reduce-javascript-payloads-with-code-splitting)
```

Comment:
This should be "code splitting", correct?

Reply:
Good catch